yarn.lock

```diff
@@ -0,0 +1,3571 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Can you describe what changes you made to the yarn file?

Reply:
cc @KayzeTheGeek

Reply:
No changes was made